### ISSUE TITLE
Intern + supertest is having issues with the version of async being used

### ISSUE THREAD

**Issue by brakon** [krakenjs/findatag#12]
If you create a test with supertest and intern (https://theintern.github.io/intern/) it will give the following exception:

```
TypeError: undefined is not a function
  at Parser.<anonymous>  <node_modules/engine-munger/node_modules/localizr/node_modules/findatag/lib/parseStream.js:87:15>
```

Test code in the test would look similar too:

```
request(mock).get('/').expect(200).end(function(err, res) {
                    if (err) {
                        reject(err);
                    } else {
                        resolve(err);
                    }
                });
```

where mock is in the lines of mock = app.listen(1337);

The issue here would be solved by updating the version of async, which Findatag uses an old version (async v0.2).
The reason of the breakage is because inside an intern test the keyword "define" does exist, which makes the async module use AMD way of exposing the module, since it checks if current call is AMD before CommonJS. In new versions of Async they check for CommonJS (module.exports) first, and then AMD (define)

See commit: https://github.com/caolan/async/commit/db06259ab373e669059ae126c4403cf0e6f9af52

**Comment by aredridel** [comment]
Alright then! Which version of `findatag` are you using?

**Comment by brakon** [comment]
Well engine-munger is the one using findatag :)
Using engine-munger v0.2.5 which uses localizr v1.0.4 which uses findatag v0.0.9 which uses async v0.2

This is what the current kraken yeoman generator sets up.

**Comment by aredridel** [comment]
Yup! I just ask because a lot of these modules have had version bumps.

**Comment by aredridel** [comment]
This should be fixed in findatag 0.0.10 and 1.0.1.

**Comment by aredridel** [comment]
And you may want to watch https://github.com/krakenjs/localizr/pull/17
